roles/rsyslog/tasks/outputs/elasticsearch/main.yml

```diff
@@ -27,7 +27,10 @@
 
     - name: Check certs - use_cert is true, but triplets are not given
       fail:
-        msg: "Error: you specified use_cert: true; you must specify all 3 of ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src, private_key_src, or set use_cert: false in the elasticsearch output named {{ item.name }}"
+        msg: "Error: you specified use_cert: true; you must specify all 3 of \
```

Comment:
I suggest using the yaml multiline string operator `>-` here:
```yaml
msg: >-
  Error: you specified use_cert: true; you must specify all 3 of
  ca_cert, cert, private_key, or all 3 of ca_cert_src, cert_src,
  private_key_src, or set use_cert: false in the elasticsearch output
  named {{ item.name }}
```
the `>` will concatenate all of the lines into a single line for output, with each newline in the input being replaced by a space in the output - the `-` means do not add a trailing newline or space to the output